test/aws/files/01_service.yml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
```

Comment:
If there's any chance we'll drop more stuff in here should we name them 01_ssh_service.yml, 02_ssh_serviceaccount.yml, etc?

Reply:
Right, `oc apply` applies files in alphabetical order, so first the namespace has to be created, then a service etc.